src/viam/ur/module/ur_arm.cpp

```diff
@@ -573,8 +591,16 @@ void URArm::move(std::vector<Eigen::VectorXd> waypoints, std::chrono::millisecon
         if (std::isinf(duration)) {
             throw std::runtime_error("trajectory.getDuration() was infinite");
         }
+        if (std::isnan(duration)) {
```

Comment:
You could maybe collapse to `isfinite`, though you would lose the distinction between the cases unless you used `fpclassify` to synthesize the message.

Reply:
To be clear, by collapse i mean "combine the calls to `isinf` and `isnan`"